manifests/init.pp

```diff
@@ -83,7 +83,7 @@
     Class['passenger::config']
   }
 
-  if type($include_build_tools) == 'string' {
+  if type3x($include_build_tools) == 'string' {
```

Comment:
Would it be better to use `is_string` to avoid calling a deprecated function like `type3x`?

For example:

```Puppet
  if is_string($include_build_tools) {
  ...
```